O365DataRetriever.ps1

```diff
@@ -703,7 +706,7 @@ $ConnectButton.Add_Click( {
             $ConnectButton.Foreground = "White"
             $ConnectButton.Content = "Connected"
             $ConnectButton.IsHitTestVisible = $false
-            $AdminPwdTextbox.Clear()
+            #$AdminPwdTextbox.Clear()
```

Comment:
Clearing the password box is only not to leave the asterisks on display. It has never impacted the rest of the script. Needs more investigation with VSCode or more issues about this before changing it.